go/vm/world_state.go

```diff
@@ -33,6 +33,7 @@ type WorldState interface {
 	SetStorage(Address, Key, Word) StorageStatus
 
 	SelfDestruct(addr Address, beneficiary Address) bool
+	SelfDestruct6780(addr Address, beneficiary Address) bool
```

Comment:
As discussed on slack ... this might not even be necessary. This entire PR could probably be skipped.